ruby/private/binary_wrapper.tpl

```diff
@@ -122,28 +117,10 @@ def main(args)
       end
     end
   end
-  exec(ruby_program, '--disable-gems', *rubyopt, main, *args)
+  exec(ruby_program, *rubyopt, main, *args)
```

Comment:
Yes, I also didn't understand why we need to disable ruby gems.

Speaking of ruby gems — when you install a brand new Ruby interpreter, remember to run `gem update --system` — this only needs to run once because rubygems rarely get upgraded, but with Ruby 2.6 this will bring Bundler version 2 as part of RubyGems.